tests/components/zwave_js/test_climate.py

```diff
@@ -88,6 +89,7 @@ async def test_thermostat_v2(
     )
 
     client.async_send_command.reset_mock()
+    set_value_response(client, True)
```

Comment:
Can't we set a default return value in the client fixture?

Reply:
we just reset the mock above which is why we have to set the return value again

Reply:
`reset_mock` doesn't clear return values.

https://docs.python.org/3/library/unittest.mock.html#unittest.mock.Mock.reset_mock